pkg/products/amqonline/reconciler.go

```diff
@@ -350,3 +359,43 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcilePrometheusRule(ctx context.Context, installation *integreatlyv1alpha1.Installation, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-sli",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{}
+	rules = append(rules, monitoringv1.Rule{
+		Alert: fmt.Sprintf("AMQ-SLI-1.1: AMQ Online console is not available in namespace %s", r.Config.GetNamespace()),
```

Comment:
Is there a naming standard for the alerts/messages that would be better suited for these fields?
In Prometheus the `alert` seems to be a short string while `message` is more descriptive but we have it the other way around here